src/guacscanner/guacscanner.py

```diff
@@ -0,0 +1,548 @@
+"""Query AWS for new (destroyed) instances and add (remove) Guacamole connections for them.
+
+Also check for instances that have been destroyed and remove their
+corresponding connections.
+
+EXIT STATUS
+  0   Update was successful.
+  >0  An error occurred.
+
+Usage:
+  guacscanner [--log-level=LEVEL] [--oneshot] [--sleep=SECONDS] [--postgres-password=PASSWORD|--postgres-password-file=FILENAME] [--private-ssh-key=KEY|--private-ssh-key-file=FILENAME] [--postgres-username=USERNAME|--postgres-username-file=FILENAME] [--rdp-password=PASSWORD|--rdp-password-file=FILENAME] [--rdp-username=USERNAME|--rdp-username-file=FILENAME] [--vnc-password=PASSWORD|--vnc-password-file=FILENAME] [--vnc-username=USERNAME|--vnc-username-file=FILENAME] [--vpc-id=VPC_ID]
```

Comment:
Shouldn't `postgres-username/file` come immediately after `postgres-password/file`?
```suggestion
  guacscanner [--log-level=LEVEL] [--oneshot] [--sleep=SECONDS] [--postgres-password=PASSWORD|--postgres-password-file=FILENAME] [--postgres-username=USERNAME|--postgres-username-file=FILENAME] [--private-ssh-key=KEY|--private-ssh-key-file=FILENAME] [--rdp-password=PASSWORD|--rdp-password-file=FILENAME] [--rdp-username=USERNAME|--rdp-username-file=FILENAME] [--vnc-password=PASSWORD|--vnc-password-file=FILENAME] [--vnc-username=USERNAME|--vnc-username-file=FILENAME] [--vpc-id=VPC_ID]
```

Reply:
I believe this is a duplicate suggestion.

Reply:
Oops, my bad. 🤦